package.json

```diff
@@ -40,6 +40,10 @@
 		"docker:ngrok": "yarn docker:compose -f docker/docker-compose-ngrok.yml",
 		"docker:ngrok-up": "yarn docker:ngrok up",
 		"docker:ngrok-down": "yarn docker:ngrok down",
+		"docker:jt": "echo 'The tool is only available to Automatticians, see `docker/README.md` for more information.",
+		"docker:jt-up": "[ -f ./docker/bin/jt/tunnel.sh ] || { echo 'Use `Tunneling scripts are not installed.'; exit; }; ./docker/bin/jt/tunnel.sh",
+		"docker:jt-down": "[ -f ./docker/bin/jt/tunnel.sh ] || { echo 'Use `Tunneling scripts are not installed.'; exit; }; ./docker/bin/jt/tunnel.sh break",
+		"docker:jt-config": "[ -f ./docker/bin/jt/config.sh ] || { echo 'Use `Tunneling scripts are not installed.'; exit; }; ./docker/bin/jt/config.sh",
```

Comment:
The message printed by these is weird, the "Use `" seems left over from an earlier version.

The message also isn't all that informative. I might change it to something like `Tunneling scripts are not installed. See the section "Jurassic Tube Tunneling Service" in docker/README.md.` so people have an easier time of finding out how to install the scripts.

Reply:
That sounds a lot better, thanks.